error/runtime_spec.go

```diff
@@ -30,7 +30,7 @@ const (
 	RootOnNonHyperV
 	// RootOnHyperV represents the error code of root setting test on hyper-v containers
 	RootOnHyperV
-	// PathFormatOnWindows represents the error code of the path format test on Window
+	// PathFormatOnWindows represents the error code of the path format test on Windows
 	PathFormatOnWindows
 	// PathName represents the error code of the path name test
 	PathName
```

Comment:
This could be more specific, e.g. `PosixRootPathConvention` or `PosixRootPathRootfs`, because it's a POSIX-only SHOULD about `root.path` suggesting “[the conventional `rootfs`][1]”.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0/config.md#root